utils/main.go

```diff
@@ -102,29 +110,40 @@ func ValidateUserKey(user string) error {
 	return nil
 }
 
-// GetUserKey takes a string and determines if it is a valid SSH key and returns
-// the user and key if valid, nil otherwise.
+// GetUserKey returns a user and a SSH key if a rawKey has a correct format, nil otherwise.
+// It doesn't validate entries.
 func GetUserKey(rawKey string) (string, string, error) {
-	key := strings.Trim(rawKey, " ")
-	if key == "" {
+	tmpKey := strings.Trim(rawKey, " ")
```

Comment:
Why do you need tmp key here? We're chaining the string transformation.

Reply:
You're right, I'll rewrite it.

Reply:
Done.